recipes/vmd/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "vmd" %}
+{% set version = "1.9.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://www.ks.uiuc.edu/Research/vmd/vmd-1.9.3/files/final/vmd-1.9.3.bin.LINUXAMD64-CUDA8-OptiX4-OSPRay111p1.opengl.tar.gz
+  sha256: 145b4d0cc10b56cadeb71e16c54ab8be713e268f11491714cd617422758ec643
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+  script:
+    - export VMDINSTALLBINDIR=$PREFIX/bin
+    - export VMDINSTALLLIBRARYDIR=$PREFIX/lib
+    - ./configure LINUXAMD64
+    - ./configure
```

Comment:
What does the second call to `configure` do?

Reply:
The first `configure` only sets an option.